src/uikit.cs

```diff
@@ -2063,6 +2063,101 @@ interface UIObjectRestoration {
 #endif
 	}
 
+	interface IUIViewAnimating {}
+
+	[iOS(10,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
```

Comment:
`[BaseType]` is only needed when `[Model]` is used, otherwise code generation is incorrect (needs to be fixed). see https://github.com/xamarin/xamarin-macios/pull/231